submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/helpers/FireFilesSource.kt

```diff
@@ -24,7 +24,8 @@ class FireFilesSource(
     ): ExtFile? {
         return when (dbFile) {
             null -> null
-            else -> fireClient.findByDb(dbFile, path, attributes)
+            is ConfiguredDbFile -> null
```

Comment:
Following the thread above, we wouldn't really need this distinction since this source would only contain FIRE files and the other DbFiles (bypassed files)